sufia-models/lib/sufia/models/utils.rb

```diff
@@ -1,12 +1,26 @@
+# This module includes a utility methods
 module Sufia
   module Utils
     extend ActiveSupport::Concern
 
+    # retry the block if the conditional call is true unless we hit the maximum tries
+    #
+    # @param number_of_tries [Fixnum] maximum number of times to retry the block
```

Comment:
This method doesn't expect a Fixnum for number_of_tries but rather an Enumerator, so perhaps `s/Fixnum/#each/`